extensions/ql-vscode/package.json

```diff
@@ -967,7 +967,7 @@
         },
         {
           "command": "codeQLVariantAnalysisRepositories.importFromCodeSearch",
-          "when": "view == codeQLVariantAnalysisRepositories && viewItem =~ /canImportCodeSearch/",
+          "when": "view == codeQLVariantAnalysisRepositories && viewItem =~ /canImportCodeSearch/ && config.codeQL.codeSearch",
```

Comment:
This is cool. Not sure how I hadn't realised this but I didn't know you could reference settings directly from `package.json`.

In this case do we need the changes to `config.ts` at all? The `CODE_SEARCH` constant, and `showCodeSearch` function are only there so we can reference the setting from within our own code, but we don't currently need to do that.

Reply:
Indeed, my mistake